lib/Doctrine/DBAL/DBALException.php

```diff
@@ -156,7 +156,7 @@ public static function driverExceptionDuringQuery(Driver $driver, Throwable $dri
 
         $msg .= ":\n\n" . $driverEx->getMessage();
 
-        return static::wrapException($driver, $driverEx, $msg);
+        return self::wrapException($driver, $driverEx, $msg);
```

Comment:
💡 PHPStan now complains if we invoke a private method through LSB because that is a potentially unsafe call.

Reply:
I have no idea why it was `static` in the first place.